### DISCUSSION TITLE
Storybook and Angular 10

### DISCUSSION THREAD

**Discussion by SimonSch** [storybookjs/storybook#11327] [General]
Hi, I just read that angular 10 has been released and wanted to ask if anyone has already tried storybook out with it, and if their are any known issues.

**Comment by shilman** [comment]
cc @kroeder @matheo

**Comment by Marklb** [comment]
I have been using Angular `10.0.1` with Storybook `6.0.0-beta.38` and it seems to be mostly working fine. Most of my problems have been from not keeping up with changes like I should have. I updated from Angular `8.2.14` and Storybook `5.3.0-beta.16` to Angular `10.0.1` and Storybook `6.0.0-beta.38` and most problems were nothing I couldn't easily figure out. The main problem I had was assets not resolving, but I found that problem and until it is fixed the object syntax can be used, which I described [here](https://github.com/storybookjs/storybook/issues/11152#issuecomment-653438373).
